content/Guides/DistributedComplianceLedger.md

```diff
@@ -0,0 +1,142 @@
+| Compliance status    | Information about the certification status including certification ID, certification date, etc. | Data is entered by the Alliance's Certification Center account.|
+| PAA Root CAs         | Information about Product Attestation Authorities (PAA) Root CAs used by commissioners during Device Attestation process. | Data is proposed by the Alliance's Trustee and approved by other DCL Trustees.|
+
+Additional information available in the [Distributed Compliance Ledger (DCL) Policies, Procedure and Governance](https://groups.csa-iot.org/wg/members-all/document/26075).
+
+### Types of nodes that operate in the DCL
+
+| Type of Node         | Description                                                                                                                                          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Validator Node (VN)  | It participates in creating a "consensus" process to add information into the ledger.                                                                |
+| Sentry Node (SN)     | It doesn\'t participate in consensus and wraps the VN representing it for the rest of the network as one of the ways for DDoS and access protection. |
+| Observer Node (ON)   | It doesn\'t participate in consensus and is optimized for public data read and authenticated write interactions with the ledger.                     |
+
+### Types of account roles that can interact with the DCL
+
+| Account Role         | Description                                                                                                                                          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Trustee              | It is entrusted to approve or deny other DCL roles and PAA Root CAs in the ledger, as well as to disable Validator Nodes.                            |
+| Node Admin           | It can instantiate a single Validator Node.                                                                                                          |
+| Vendor               | It can write vendor and product information to the ledger. Alliance members can have one or more vendor accounts.                                    |
+| Vendor Admin         | It can add and update the Vendor information table of the ledger.                                                                                    |
+| Certification Center | It can submit, update, delete or revoke certification status of a product based on the Certification application in the Certification Tool.          |

```

Comment:
Drop the "It is/can" for each description

"Entrusted to approve or deny..."
"Instantiate a single..."
"Write vendor and product..."

Reply:
Done